packages/webdriverio/tests/commands/element/dragAndDrop.test.js

```diff
@@ -134,4 +154,23 @@ describe('dragAndDrop', () => {
         expect(got.mock.calls[4][1].json).toEqual({ element: null, xoffset: 123, yoffset: 321 })
         expect(got.mock.calls[5][1].uri.pathname).toContain('/foobar-123/buttonup')
     })
+
+    it('should do a dragAndDrop with the given co-ordinates and duration(no w3c)', async () => {
+        const browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar-noW3C'
+            }
+        })
+
+        const elem = await browser.$('#foo')
+        await elem.dragAndDrop({ x: 123, y: 321 }, { duration: 200 })
+
+        expect(got.mock.calls[2][1].uri.pathname).toContain('/foobar-123/moveto')
+        expect(got.mock.calls[2][1].json).toEqual({ element: 'some-elem-123' })
+        expect(got.mock.calls[3][1].uri.pathname).toContain('/foobar-123/buttondown')
+        expect(got.mock.calls[4][1].uri.pathname).toContain('/foobar-123/moveto')
+        expect(got.mock.calls[4][1].json).toEqual({ element: null, xoffset: 123, yoffset: 321 })
+        expect(got.mock.calls[5][1].uri.pathname).toContain('/foobar-123/buttonup')
```

Comment:
we do all these assertions already in the test before so there is no need to have them here again, we should rather check if this action took 200ms

Reply:
Sure. Can do that. 
But, rather than the default way of using Javascript to measure, I am unaware if any other options exist, so had left it for review :-) Are there any options by default available to do the same, if you can please guide me here.

Reply:
Got it. Had a confusion. Making this change.

Reply:
Done.